### PR TITLE
Add suru to kubeflow pages

### LINKED ISSUE #6453: Add suru to all Kubeflow pages
Add suru strips from https://ubuntu.com/_docs/strips to the following pages:

- [x] `p-strip--suru-image` to https://ubuntu.com/kubeflow

- [x] `p-strip--suru-topped` to https://ubuntu.com/kubeflow/features

- [x] `p-strip--suru-image` to https://ubuntu.com/kubeflow/install

- [x] ` p-strip--suru-bottomed` to https://ubuntu.com/kubeflow/consulting

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add suru to top of Kubeflow pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubeflow
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the Kubeflow homepage and all it's subpages have a suru at the top of the page

## Issue / Card

Fixes #6453

### CODE REVIEW

File: templates/kubeflow/features.html [hunk 1]
```diff
@@ -7,17 +7,19 @@
...
-      <p class="p-heading--three">Architectural freedom. Fully automated operations. Accelerated Deep Learning</p>
-      <p>Canonical&rsquo;s AI solutions such as <a href="/kubeflow/install">Kubeflow on Ubuntu</a> use your existing on-premise clusters and GPGPUs efficiently, giving you architectural freedom with storage and networking while sharing operational code with a large community. From your developer workstation, to your racks, to the public cloud, AI on Ubuntu is accelerated with the latest tools, drivers and libraries.</p>
-      <p>The standard for enterprise machine learning, from Silicon Valley to Wall Street, for the Fortune 50 and for startups.</p>
-      <p><a class="js-invoke-modal" href="/kubeflow/contact-us?product=ai-features">Contact us for machine learning, deep learning and AI consulting&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-4 col-start-large-9 u-vertically-center u-align--center u-hide--small">
-      <img src="https://assets.ubuntu.com/v1/7d33e0a1-AI+illustration+hero.svg" alt="Kubeflow" width="323" />
+<section class="p-strip is-bordered u-no-padding--top u-no-padding--bottom">
```
Comment (solazio) [inline templates/kubeflow/features.html]: You can add the `p-strip--suru-topped` to line 10 and remove `p-strip u-no-padding--top u-no-padding--bottom`. And also you can remove the next level `div` (line 11)
```suggestion
<section class="p-strip--suru-topped is-bordered ">
```

File: templates/kubeflow/index.html [hunk 2]
```diff
@@ -6,16 +6,18 @@
...
-  <div class="row">
-    <div class="col-7">
-      <h1>AI / ML from workstation to production</h1>
-      <p>Ubuntu is the platform to power your Artificial Intelligence ambitions &mdash; from developer workstations, to racks, to clouds and to the edge with smart connected IoT.</p>
-      <p>Canonical&rsquo;s Kubeflow supports the most popular tools for machine learning &mdash; starting with JupyterHub and Tensorflow &mdash; in a standardised workflow running on Kubernetes.</p>
-      <p>
-        <a class="p-button--positive" href="/kubeflow/install">Get started by installing Kubeflow</a>
-        <a class="p-button--neutral js-invoke-modal" href="/kubeflow/contact-us?product=ai-index">Call the experts</a>
-      </p>
+<section class="p-strip is-deep is-bordered u-no-padding--top u-no-padding--bottom">
```
Comment (solazio) [inline templates/kubeflow/index.html]: You can add the p-strip--suru-topped to line 9 and remove p-strip u-no-padding--top u-no-padding--bottom. And also you can remove the next level div (line 10)

File: templates/kubeflow/install.html [hunk 3]
```diff
@@ -6,14 +6,16 @@
...
 {% block content %}
-<section class="p-strip--light is-deep is-bordered">
-  <div class="row">
-    <div class="col-8">
-      <h1>Install Kubeflow on Ubuntu</h1>
-      <p>Create and train machine learning models on your laptop, in your data center, or in the cloud.</p>
-    </div>
-    <div class="col-3 col-start-large-10 u-align--center u-hide--small">
-      <img src="https://assets.ubuntu.com/v1/1ec1605a-kubeflow.svg?w=275" alt="" width="275" />
+<section class="p-strip is-deep is-bordered u-no-padding--top u-no-padding--bottom">
```
Comment (solazio) [inline templates/kubeflow/install.html]: You can add the p-strip--suru-topped to line 9 and remove p-strip u-no-padding--top u-no-padding--bottom. And also you can remove the next level div (line 10)

File: templates/kubeflow/features.html (hunk 1 above)
Comment (steverydz) [reply templates/kubeflow/features.html]: Done

File: templates/kubeflow/install.html (hunk 3 above)
Comment (steverydz) [reply templates/kubeflow/install.html]: Done

**Comment by steverydz** [comment]
> @steverydz just a few minor comments in the code

Thanks @solazio. All fixed.

**Pull request merged**